src/features/copyVariableName.ts

```diff
@@ -7,14 +7,25 @@ export const registerCopyVariableName = () => {
         if (!activeEditor || activeEditor.viewColumn === undefined) return
         // also would make sense to support mutliple selections/cursors
         const lineText = activeEditor.document.lineAt(activeEditor.selection.end).text
-        const regexps = [
-            /\s*(?:const(?: {)?|let(?: {)?|type|interface|import(?: {)?|function) ([\w\d]+)/,
-            /\s*(?:\.|(?:.*(?:class|className)=["']))([\w\d-]+?)\s*[{"']/,
-        ]
+        const regexps = {
+            variableNameRegex: /\s*(?:const(?: {)?|let(?: {)?|type|interface|import(?: {)?|function) ([\w\d]+)/,
+            classContentRegex: /\s*(?:\.|(?:.*(?:class|className)=["']))([\w\d\s-]+?)\s*[{"']/,
+        }
         let varName: string | undefined
-        for (const regexp of regexps) {
+        for (const [regexpName, regexp] of Object.entries(regexps)) {
             varName = regexp.exec(lineText)?.[1]
-            if (varName) break
+            if (varName) {
+                if (regexpName === 'classContentRegex') {
+                    const classes = varName.split(' ')
+                    if (classes.length === 1) break
+                    // eslint-disable-next-line no-await-in-loop
+                    const selectedClass = await vscode.window.showQuickPick(classes, { title: 'Copy classname' })
```

Comment:
@maIIady what do you think of making it canPickMany=true e.g. so we can copy multiple clases at once?

Reply:
If it doesn't make sense feel free to merge

Reply:
Don't see a googd reason to add it here, but I'm currently thinking about implementing a some kind of code action for instant class creation, mb gonna use ur suggestion there